sqlengine/postgres_engine_test.go

```diff
@@ -414,6 +414,14 @@ var _ = Describe("PostgresEngine", func() {
 				connectionStringNew = postgresEngine.URI(address, port, dbname, createdUser, createdPassword)
 			})
 
+			AfterEach(func() {
```

Comment:
I think this has been done now (though I'm lacking some context on the original comment).